packages/picasso/src/RichText/hooks/useRichText/useRichText.tsx

```diff
@@ -64,18 +64,27 @@ const picassoMapper = (child: ReactNode): ReactNode => {
 
   const type =
     componentMap[child.type as keyof typeof componentMap] || child.type
-  const mappedChildren = child.props.children.map(picassoMapper)
+
+  const hasChildChildren = child.props.children
```

Comment:
```suggestion
  const hasChildChildren = Boolean(child.props.children)
```
it will work even without this, but we do this for a better readability almost everywhere in the code